extension/src/implementation.ts

```diff
@@ -0,0 +1,52 @@
+import { Installation } from "@exasol/extension-manager-interface";
+import { ExaAllScriptsRow } from "@exasol/extension-manager-interface/dist/exasolSchema";
+import { ADAPTER_SCRIPT_NAME, Context, IMPORT_SCRIPT_NAME } from "./common";
+
+function findScriptByName(scripts: ExaAllScriptsRow[], name: string): ExaAllScriptsRow | undefined {
+    return scripts.find(script => script.name === name);
+}
+
+export function findInstallations(scripts: ExaAllScriptsRow[], context: Context): Installation[] {
+    const importScript = findScriptByName(scripts, IMPORT_SCRIPT_NAME);
+    const adapterScript = findScriptByName(scripts, ADAPTER_SCRIPT_NAME);
+    if (isValidImportScript(importScript) && isValidAdapterScript(adapterScript)) {
+        return [{
+            name: `${adapterScript.schema}.${adapterScript.name}`, version: context.version, instanceParameters: []
+        }];
+    } else {
+        return [];
+    }
+}
```

Comment:
I do not understand the role of parameter `context` here.
Maybe you could explain to me?